cdap-docs/admin-manual/source/installation/cloudera.rst

```diff
@@ -76,6 +76,41 @@ These roles map to the :ref:`CDAP components <admin-manual-cdap-components>` of
 .. --------------------
 .. include:: ../_includes/installation/hadoop-configuration.txt
 
+#. Add additional jars to the YARN Application Classpath for Spark jobs.
+
+   .. highlight:: console
+
+   If you plan on running Spark programs from CDAP, CDAP requires that additional entries be added to
+   the YARN application classpath, as the Spark installed on Cloudera Manager clusters
+   is a "Hadoop-less" build and does not include Hadoop jars required by Spark.
+
+   To resolve this, generate the correct classpath by going to a host in your cluster that
+   has a Spark Gateway, and go to the Spark configuration directory, located either in::
+ 
+     /opt/cloudera/parcels/CDH/lib/spark/conf
+   
+   or::
+ 
+     /etc/spark/conf
+ 
+   From within that, run this command to create a file with the additional entries
+   required::
+
+     $ cat classpath.txt | grep "jar$" | grep -v "^.*jars\/hive-exec-" | paste -sd, - > add_classpath.txt
+ 
+   (This command removes the ``hive-exec`` JAR with a Guava dependency that will not work with CDAP,
```

Comment:
this command doesn't remove the jar file, while generating the new classpath file based on existing classpath file, it just ignores the hive jar. description should be clear.